From 44832890435e0140dc94061aa23dbaeeedd15a12 Mon Sep 17 00:00:00 2001
From: Huang zhibao <hzb@rock-chips.com>
Date: Fri, 30 Jun 2017 09:14:11 +0800
Subject: [PATCH 02/13] ARM64: dts: rk322xh: set aclk_vop freq to 500M

Change-Id: I3b89d1e0fa20c853b69f0a84abb79021b3da1e6c
Signed-off-by: Huang zhibao <hzb@rock-chips.com>
(cherry picked from commit 32bf9aafdf96e82452a9872cd2f48563c2b31969)

Conflicts:
	arch/arm64/boot/dts/rk322xh.dtsi
Signed-off-by: tug <tug@skyth-tek.com>
---
 arch/arm64/boot/dts/rk322xh.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rk322xh.dtsi b/arch/arm64/boot/dts/rk322xh.dtsi
index 3d40d6e..0ec135b 100644
--- a/arch/arm64/boot/dts/rk322xh.dtsi
+++ b/arch/arm64/boot/dts/rk322xh.dtsi
@@ -354,7 +354,7 @@
 			<&hclk_peri 75000000>, <&pclk_peri 75000000>,
 			<&aclk_vio 300000000>, <&hclk_vio 100000000>,
 			<&aclk_rga 300000000>, <&clk_rga 200000000>,
-			<&aclk_vop 400000000>, <&aclk_rkvdec 500000000>,
+			<&aclk_vop 500000000>, <&aclk_rkvdec 500000000>,
 			<&aclk_rkvenc 200000000>, <&aclk_vpu 300000000>,
 			<&clk_vdec_cabac 300000000>, <&clk_vdec_core 200000000>,
 			<&clk_venc_core 200000000>, <&clk_venc_dsp 100000000>,
-- 
1.7.9.5

